bundles/net.tourbook/src/net/tourbook/ui/views/tourMarker/TourMarkerView.java

```diff
@@ -467,6 +472,29 @@ private void createUI_10_TableViewer(final Composite parent) {
 //      table.setLinesVisible(_prefStore.getBoolean(ITourbookPreferences.VIEW_LAYOUT_DISPLAY_LINES));
       table.setLinesVisible(false);
 
+      table.addKeyListener(new KeyListener() {
+
+         @Override
+         public void keyPressed(final KeyEvent e) {
+
+            if (e.keyCode == SWT.DEL) {
+
+               if (_actionDeleteTourMarkers == null) {
```

Comment:
I think it is not necessary to check agains `null` but it should check if `_actionDeleteTourMarkers` is enabled or not